build/storage/recipes/README.md

```diff
@@ -6,8 +6,10 @@ different machines and demonstrates virtio-blk hot-plug to a running host.
 * [fio](./fio.md) - extends `hot-plug` scenario and describes how to run
 fio traffic by means of `host-target` container through a hot-plugged virtio-blk
 device.
+* [scale-out](./scale-out.md) - a very similar to hot-plug recipe, but describing
```

Comment:
similar to hot-plug recipe but describing
how to hot-plug/hot-unplug 64 virtio-blk devices.